src/components/profilePage/RatingStar/RatingStarStyles.css

```diff
@@ -0,0 +1,20 @@
+.header{
+    font-style: normal;
+}
+#star{
+    width: 100%;
+    display:inline-block;
+    text-align: center;
+
+}
+#rating > span {
+  display: inline-block;
+  position: relative;
+}
+#rating:hover > span:before{
+    content: url(../../../utils/Rating_Icons/star_1.png); /* Place a solid star over all the empty stars when you hover them. */
```

Comment:
Instead of using images for starts use font-awesome stars. It will reduce load on the webpage. Reviewed by me & @The-Atul-Sharma